mlflow/pipelines/utils/step.py

```diff
@@ -105,15 +106,28 @@ def _get_pool_size():
     return 1 if "PYTEST_CURRENT_TEST" in os.environ and os.name == "nt" else 0
 
 
-def get_pandas_data_profile(data_frame, title: str):
-    """Returns a data profiling object over input data frame.
+def get_pandas_data_profiling_comparison_view(inputs: Iterable[Tuple[str, pd.DataFrame]]) -> str:
```

Comment:
Let's not call it `_comparison_view` since there's only 1 function exists (as compared to having a separate function `get_pandas_data_profile`). It is confusing when only 1 dataset is provided. `get_pandas_data_profiles` probably is sufficient.

Reply:
Sounds good :) 
Let me change it back